db/seeds.rb

```diff
@@ -1,3 +1,7 @@
 # frozen_string_literal: true
 
-User.create!(email: 'test@test.com', password: 'password')
+User.create!(email: 'test@test.com', password: 'password',
```

Comment:
Is this a solution?

User.where(email: 'test@test.com', password: 'password', role: :admin).first_or_create